cmd/cli/main.go

```diff
@@ -173,12 +174,25 @@ func main() {
 		os.Exit(1)
 	}
 
+	var flinkAPITimeoutSeconds int64
+	if len(os.Getenv("FLINK_API_TIMEOUT_SECONDS")) > 0 {
```

Comment:
Maybe I'm just bitching now but seeing that it's quite a bit of code with some logic in it, can we move it to a separate func and add a test too?

Reply:
done